app/views/case_information/new.html.erb

```diff
@@ -19,8 +19,8 @@
           <%= label_tag "case_information[case_allocation]", "National Probation Service (NPS)", class: "govuk-label govuk-radios__label" %>
         </div>
         <div class="govuk-radios__item">
-          <%= radio_button_tag("case_information[allocation]", "CRC", false, class: "govuk-radios__input") %>
-          <%= label_tag "case_information[allocation]", "Community Rehabilitation Company (CRC)", class: "govuk-label govuk-radios__label" %>
+          <%= radio_button_tag("case_information[case_allocation]", "CRC", false, class: "govuk-radios__input") %>
```

Comment:
I'm unsure how this fixes the problem with Tier C & D ...

Reply:
The controller is expecting a `case_allocation` field not `allocation`